app/routes/planet_routes.py

```diff
@@ -0,0 +1,94 @@
+from flask import Blueprint, abort, make_response, request, Response
+from app.routes.db import db
+from app.models.planet import Planet
+
+
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
```

Comment:
Convention is to name this `bp`, inside of `app/__init__.py` we could then import this under an alias to prevent name conflicts like so:
```python
from app.routes.planet_routes.py import bp as planets_bp
```